docs/docs.ipynb

```diff
@@ -39,21 +39,19 @@
   {
    "cell_type": "code",
    "execution_count": 1,
-   "metadata": {
-    "collapsed": false
-   },
+   "metadata": {},
    "outputs": [
     {
      "name": "stderr",
      "output_type": "stream",
      "text": [
-      "\"мешок\": 100%|███████████████████████████████| 10/10 [00:07<00:00,  1.40docs/s]\n"
+      "\"мешок\": 100%|██████████| 10/10 [00:01<00:00,  8.10docs/s]\n"
      ]
     },
     {
      "data": {
       "text/plain": [
-       "[Result(query=мешок, N=10, params={'numResults': 10, 'kwic': True, 'nLeft': None, 'nRight': None, 'queryLanguage': None, 'subcorpus': 'main', 'tag': False, 'start': 0, 'writingSystem': None})]"
+       "[Result(query=мешок, N=10, params={'n_results': 10, 'kwic': True, 'n_left': None, 'n_right': None, 'query_language': None, 'subcorpus': 'main', 'get_analysis': False, 'gr_tags': None, 'start': 0, 'writing_system': None})]"
```

Comment:
The attributes are now called this way

Reply:
They were renamed in the code but not in docs. Therefore no examples from here would work